.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "helpz/_posts"]
+	path = helpz/_posts
+	url = https://github.com/tuna/mirrorz-help-ng.git
```

Comment:
Please use a relative path to refer to the submodule, so that the protocol is not fixed.